docs/man_pages/publishing/appmanager.md

```diff
@@ -16,7 +16,8 @@ WARNING: This command is not applicable to mobile website projects. To view the
 * `upload android` - Builds the project for Android and uploads the binary to Telerik AppManager.
 * `upload ios` - Builds the project for iOS and uploads the binary to Telerik AppManager.
 <% if(isCordova) { %>* `upload wp8` - Builds the project for Windows Phone and uploads the binary to Telerik AppManager.
-* `livesync` - Publishes a new Telerik AppManager LiveSync update of your application.<% } %> 
+* `livesync` - Publishes a new Telerik AppManager LiveSync update of your application.
+* `groups` - Lists all available user groups from Telerik AppManager.<% } %> 
```

Comment:
I've changed this in all 'groups' option occurrences (in all files with  Related Commands that need to show 'appmanager groups')